primitives/src/transaction.rs

```diff
@@ -598,3 +598,83 @@ impl<'a> Arbitrary<'a> for Txid {
         Ok(Txid(t))
     }
 }
+
+#[cfg(feature = "alloc")]
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn transaction_functions() {
+        let txin1 = TxIn {
```

Comment:
```suggestion
        let txin = TxIn {
```

I'm guessing you had a txin2 during development?

Reply:
I did.  Changed as suggested.